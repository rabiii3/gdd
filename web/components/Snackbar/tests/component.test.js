import React from 'react';
import { shallow } from 'enzyme';
import { Component } from '..';
import { IntlProvider } from 'react-intl';

const intlProvider = new IntlProvider({ locale: 'en', messages: {} }, {});
const { intl } = intlProvider.getChildContext();

describe('web | components | Snackbar | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        actions: {},
        classes: {},
        error: { type: 'TYPE' },
        intl: { formatMessage: jest.fn() },
      };
      const wrapper = shallow(<Component {...props} intl={intl}  />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
