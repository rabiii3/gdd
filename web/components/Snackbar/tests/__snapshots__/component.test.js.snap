// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | Snackbar | component render should match snapshot 1`] = `
<WithStyles(Snackbar)
  ContentProps={
    Object {
      "aria-describedby": "message-id",
    }
  }
  anchorOrigin={
    Object {
      "horizontal": "left",
      "vertical": "bottom",
    }
  }
  autoHideDuration={6000}
  onClose={[Function]}
  open={true}
  variant="error"
>
  <MySnackbarContent
    classes={Object {}}
    message="TYPE"
    onClose={[Function]}
    variant="error"
  />
</WithStyles(Snackbar)>
`;
